DEVELOPERS.md

```diff
@@ -37,16 +37,22 @@ just run  # run python server
 
 ## Test data
 
-`data/test-nursery.py` will use ACRO to generate some test outputs in `outputs/`
+The script `data/test-nursery.py` uses ACRO to generate outputs from a public test
+dataset. It will generate some test outputs in `outputs/`.
 
-This should be generated for you automatically by just rules, but if you need
-to manually regenerate it:
+It can be run with the following command:
 
 ```
-just clean 
 just test-outputs
 ```
 
+The required data should be downloaded for you automatically via the above command, but if
+you need to manually regenerate it, run:
+
+```
+just clean
+just test-data
```

Comment:
```suggestion
just test-outputs
```

test-data is different, and test-outputs calls it anyway

Reply:
That's not right, just clean is needed in the case of regeneration, which is why I added it, as it already bit Tom.

`just test-outputs` will regenerate the data iff data/test-nursery has changes. If we add new files (e.g. the XandY.jfif), it won't regenerate (as thats hard to detect)

The easiest way to force clean regeneration of the test outputs is to run `just clean` first.